appveyor.yml

```diff
@@ -99,7 +99,7 @@ build_script:
 
 test_script:
 - cd %ROOT_DIRECTORY%\test
-- b2.exe -l400 --hash warnings-as-errors=on -j2 %compiler% address-model=%model% debug-iterators=on picker-debugging=on invariant-checks=full variant=%variant% %linkflags% %include% link=shared ssl=%ssl% crypto=%crypto% win-tests
+- appveyor-retry b2.exe -l400 --hash warnings-as-errors=on -j2 %compiler% address-model=%model% debug-iterators=on picker-debugging=on invariant-checks=full variant=%variant% %linkflags% %include% link=shared ssl=%ssl% crypto=%crypto% win-tests
```

Comment:
yes, it worries me too... well, I imagine the "retry" can/should be removed at some point in the future